int8 num1
int8 num2
